### PR TITLE
AT_06.02.004 verify the description can be added to a created user

### PR DESCRIPTION
**Pull request by kgold2018**
https://trello.com/c/v3ua84aD/1112-at0602004-people-add-a-description-to-a-created-user

### CODE REVIEW

File: cypress/e2e/peopleAddDescriptionToCreatedUser.cy.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+/// <reference types='cypress'/>
+
+describe("Add description to a created user", () => {
```
Comment (Rom-T) [inline cypress/e2e/peopleAddDescriptionToCreatedUser.cy.js:3]: Please name the describe as the name of your file.

File: cypress/e2e/peopleAddDescriptionToCreatedUser.cy.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  beforeEach("Create a user from Manage Jenkins>Manage Users", function () {
+    cy.get('a[href="/manage"]').click();
+    cy.get('.jenkins-section__item a[href="securityRealm/"]').click();
+    cy.get(".jenkins-app-bar__controls").click();
+    cy.get("input#username").type("created1");
+    cy.get('input[name="password1"]').type("123");
+    cy.get('input[name="password2"]').type("123");
+    cy.get('input[name="email"]').type("qa@test.com");
```
Comment (Rom-T) [inline cypress/e2e/peopleAddDescriptionToCreatedUser.cy.js:11]: Please move `data` in the fixtures

File: cypress/e2e/peopleAddDescriptionToCreatedUser.cy.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    cy.get('button[name="Submit"]').click();
+    cy.get("#breadcrumbBar").contains("Dashboard").click();
```
Comment (Rom-T) [inline cypress/e2e/peopleAddDescriptionToCreatedUser.cy.js:13]: Please move `Dashboard` in the fixtures

File: cypress/e2e/peopleAddDescriptionToCreatedUser.cy.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+  });
+
+  it("AT_06.02.004 verify the description can be added to a created user", () => {
+    cy.get('a[href="/asynchPeople/"]').click();
+    cy.get('#person-created1 a[href = "/user/created1/"]').click();
+    cy.get("#description div").not("div[class]").should("be.empty");
+    cy.get("#description-link").click();
+    cy.get("textarea[name='description']").type(
+      "Created first time a new description"
```
Comment (Rom-T) [inline cypress/e2e/peopleAddDescriptionToCreatedUser.cy.js:22]: Please move `Created first time a new description` in the fixtures

File: cypress/e2e/peopleAddDescriptionToCreatedUser.cy.js [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+    );
+    cy.get("button[name='Submit']").click();
+    cy.get("#description div")
+      .not("div[class]")
+      .should("have.text", "Created first time a new description");
```
Comment (Rom-T) [inline cypress/e2e/peopleAddDescriptionToCreatedUser.cy.js:27]: Please move `Created first time a new description` in the fixtures

File: cypress/e2e/peopleAddDescriptionToCreatedUser.cy.js (hunk 1 above)
Comment (kgold2018) [reply cypress/e2e/peopleAddDescriptionToCreatedUser.cy.js:3]: Done

File: cypress/e2e/peopleAddDescriptionToCreatedUser.cy.js (hunk 3 above)
Comment (kgold2018) [reply cypress/e2e/peopleAddDescriptionToCreatedUser.cy.js:13]: Done

File: cypress/e2e/peopleAddDescriptionToCreatedUser.cy.js (hunk 4 above)
Comment (kgold2018) [reply cypress/e2e/peopleAddDescriptionToCreatedUser.cy.js:22]: Done

File: cypress/e2e/peopleAddDescriptionToCreatedUser.cy.js (hunk 5 above)
Comment (kgold2018) [reply cypress/e2e/peopleAddDescriptionToCreatedUser.cy.js:27]: Done

File: cypress/e2e/peopleAddDescriptionToCreatedUser.cy.js (hunk 2 above)
Comment (kgold2018) [reply cypress/e2e/peopleAddDescriptionToCreatedUser.cy.js]: Done

**Comment by kgold2018** [comment]
@Rom-T  Please review my changes.Thanks

**Review by Rom-T — APPROVED** [review]
LGTM

**Review by ArinaJur — APPROVED** [review]
Good job!

**Pull request merged**